packages/mui-material/src/Chip/Chip.js

```diff
@@ -460,6 +461,7 @@ const Chip = React.forwardRef(function Chip(inProps, ref) {
       onKeyDown={handleKeyDown}
       onKeyUp={handleKeyUp}
       ref={handleRef}
+      tabIndex={disabled ? tabIndex ?? -1 : tabIndex}
```

Comment:
I think it should be:
```suggestion
      tabIndex={disabled ? -1 : tabIndex}
```
We don't want developers to apply `tabIndex` if component is `disabled`.